src/apps/answers/service.py

```diff
@@ -366,7 +367,19 @@ async def validate_multiinformant_assessment(
             activity, flow = await asyncio.gather(activity_future, flow_future)
             if not activity and not flow:
                 raise MultiinformantAssessmentInvalidActivityOrFlow()
-
+            
+        try:
+            await CheckAccessService(self.session, self.user_id).check_applet_manager_list_access(applet_id)
+            is_admin = True
```

Comment:
Please, don't use exception as IF, in this case use this code to check if user an admin

```
        is_admin = await AppletAccessCRUD(self.session).has_any_roles_for_applet(
            respondent_subject.applet_id,
            respondent_subject.user_id,
            Role.managers(),
        )
```